style.css

```diff
@@ -409,6 +391,226 @@ ul {
   background-color: #2230d2;
 }
 
+.work {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  justify-items: center;
+  margin: 30px;
+  gap: 30px;
+}
+
+.work-section {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: space-around;
+  width: 327px;
+  height: 552px;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 16px;
+}
+
+.work-header {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+}
+
+.image-desktop {
+  display: none;
+}
+
+.cardItems {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 12px;
+}
+
+.role {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.startYear {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.pinformation {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  width: 295px;
+}
+
+.langTages {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 10px;
+  list-style-type: none;
+}
+
+.btnSeepro {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 10px;
+}
+
+.btnSee-project {
+  color: #8161cc;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  width: 124px;
+  height: 48px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 5px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  background-color: #fff;
+}
+
+.btnSee-project:hover {
+  background-color: #7c49f2;
+  color: #fff;
+}
+
+.btn-seelive-pp {
+  background-image: url(./assets/images/see-live.svg);
+  background-position: 85% center;
+  background-repeat: no-repeat;
+  display: flex;
+  justify-content: flex-start;
+  width: 142px;
+  padding: 12px 16px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  gap: 12px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+.btn-seesource-pp {
+  background-image: url(./assets/images/see-source.png);
+  background-position: 85% center;
+  background-repeat: no-repeat;
+  display: flex;
+  justify-content: flex-start;
+  width: 142px;
+  padding: 12px 16px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  gap: 12px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+.popupDisplay {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 16px;
+  gap: 12px;
+  position: fixed;
+  width: fit-content;
+  top: 20px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  margin: 16px;
+}
```

Comment:
- [ ] The popup is too wide, Please adjust it's proportions to make it look similar to figma design.
- [ ] The background should be blurred , just like in the figma template. 
- [ ] Add the icons for "See Live" and "See Source" links. 




Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/180250036-a1ceecb8-1f78-4e46-a5c4-d5f32b9f042d.png)  |  ![image](https://user-images.githubusercontent.com/60618877/180249686-d7916b7b-7a19-48cb-8c22-386ca5599981.png)

- [ ] Under the main popup heading , the sub headings are missing:
It should have sub headings like this 👇 

Kindly fix these issues and your project is good to go 👍 

![image](https://user-images.githubusercontent.com/60618877/180251820-f1d7b7bd-f949-4658-bb35-7415910e4f62.png)